CHANGELOG.md

```diff
@@ -12,79 +12,97 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
-- Fixed a rare server crash that could occur when closing an actor channel right after attaching a dynamic subobject to that actor.
-- Fixed a defect in `InstallGDK.bat` which sometimes caused it to incorrectly report `Error: Could not clone...` when repositories had been cloned correctly.
-- Actors from the same ownership hierarchy are now handled together when they are load-balanced.
+- Fixed a problem that caused load balanced cloud deployments to fail to start while under heavy load.
+- Fix to avoid using packages that are still being processed in the asynchronous loading thread.
+- Fixed a bug that sometimes caused GDK setup scripts to fail to unzip dependencies.
+- Fixed a bug where RPCs that were called before calling the `CreateEntityRequest` were not processed as early as possible in the RPC ring buffer system, resulting in startup delays on the client.
+- Fixed a crash that occurred when running a game with `nullrhi` and using `SpatialDebugger`.
+- When you use a URL with options in the command line, we now parse the Receptionist parameters correctly, using the URL if necessary.
+- Fixed a bug that occurred when creating multiple dynamic subobjects at the same time, and caused them to fail to be created on clients.
+- `OwnerOnly` components are now properly replicated when a worker instance gains authority over an Actor. Previously, they were sometimes only replicated when a value on them changed (after the worker instance had already gained authority).
+- Fixed a rare server crash that could occur when closing an Actor channel immediately after attaching a dynamic subobject to that Actor.
+- Fixed a defect in `InstallGDK.bat` that sometimes caused it to incorrectly report `Error: Could not clone…` when repositories were cloned correctly.
+- Actors from the same ownership hierarchy are now handled together when they are load balanced.
+
+## SpatialOS tooling compatibility:
+If you are using the Standard Runtime variant, note the following compatibility issues:
+- The [old Inspector](https://documentation.improbable.io/spatialos-tools/docs/the-inspector) won’t work. You must use the [new Inspector](https://documentation.improbable.io/spatialos-tools/docs/the-new-inspector) instead.
+- In the [Platform SDK in C#](https://documentation.improbable.io/sdks-and-data/docs/platform-csharp-introduction), you can’t set [capacity limits](https://documentation.improbable.io/sdks-and-data/docs/platform-csharp-capacity-limiting) or use the [remote interaction service](https://documentation.improbable.io/sdks-and-data/docs/platform-csharp-remote-interactions). You also can’t use the Platform SDK to take snapshots of cloud deployments, but we’ll fix this snapshot issue in a future release.
+- In the [CLI](https://documentation.improbable.io/spatialos-tools/docs/cli-introduction), the following commands don’t work:
```

Comment:
```suggestion
- You can't generate a snapshot. However, because old snapshots are incompatible with 0.10, you need to generate new snapshots. To do this, change your Runtime variant to Compatibility Mode (see below) to generate snapshots, and then change back to Standard. 
- In the [CLI](https://documentation.improbable.io/spatialos-tools/docs/cli-introduction), the following commands don’t work:
```